chaco/plot_graphics_context.py

```diff
@@ -34,8 +34,8 @@ def __init__(self, size_or_ary, *args, **kw):
         scale = kw.pop("dpi", 72.0) / 72.0
```

Comment:
If we can assume that `size_or_ary` is a tuple of **ints**, we can make the following change instead (IIUC the code correctly) but we can be defensive and go with the changes you propose.

```suggestion
        scale = kw.pop("dpi", 72.0) // 72.0
```

Reply:
If it's alright with you I think I will leave the current approach, as you mention, to be defensive.  The reasoning being that I think `scale_ctm` can accept floats and `size_or_ary` is what we specifically want to have integer entries.  
I would guess in practice `scale` will typically be an integer but I am not certain that would always be the case